crates/context_aware_config/src/api/default_config/handlers.rs

```diff
@@ -201,6 +172,118 @@ async fn create(
     Ok(http_resp.json(default_config))
 }
 
+#[put("/{key}")]
```

Comment:
see if we want partial updates, then change it to
```suggestion
#[patch("/{key}")]
```
otherwise remove the partial update thing and accept the all the properties as input.

Reply:
I agree but we would need to do the similar change in dimension update api and other apis too. 
Can we make that change in a seperate PR , that will keep it consistent .